cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -149,7 +148,7 @@ const UNINCLUDED_SEGMENT_CAPACITY: u32 = 3;
 /// number of blocks authored per slot.
 const BLOCK_PROCESSING_VELOCITY: u32 = 1;
 /// Relay chain slot duration, in milliseconds.
-const RELAY_CHAIN_SLOT_DURATION_MILLIS: u32 = 6000;
+pub const RELAY_CHAIN_SLOT_DURATION_MILLIS: u32 = 6000;
```

Comment:
I asked about this here: https://github.com/paritytech/polkadot-sdk/pull/2826#discussion_r1443537059